### DISCUSSION TITLE
BBN Marine OS for Raspberry Pi Discussion Forum

### DISCUSSION THREAD

**Discussion by mgrouch** [bareboat-necessities/lysmarine_gen#109] [Q&A] [answered]
BBN Marine Linux OS
2021-09-01-Release-Stable

Notable improvements: Multi-touch is now supported in OpenCPN, fixed all other touch support issues

    Two finger zoom for OpenCPN
    Touch fixes
    Mopidy fixes
    PyPilot fixes
    New RaspiOS kernel

Download at https://cloudsmith.io/~bbn-projects/repos/bbn-repo/packages/?q=lysmarine

**Comment by veleirocica** [comment, marked as answer]
Nice!

**Comment by DH34-HUB** [comment]
HELLO a password is requested in the Terminal by default it is mentioned user this does not work in the lysmarine-bbn-bullseye_2023-05-17-raspi version… who has the solution thank you?

**Reply by mgrouch** [reply]
It worked for me. What is the password you are trying with? And what is user name?

**Reply by DH34-HUB** [reply]
when I open the Terminal the username is user@lysmarine and the BBN doc suggests the password user

**Reply by DH34-HUB** [reply]
I logged in as a super user and it just worked great have a nice day
